src/Jackett/Indexers/FrenchTorrentDb.cs

```diff
@@ -0,0 +1,172 @@
+﻿using CsQuery;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Jackett.Indexers
+{
+    class FrenchTorrentDb : IndexerInterface
+    {
+        public event Action<IndexerInterface, Newtonsoft.Json.Linq.JToken> OnSaveConfigurationRequested;
+
+        public event Action<IndexerInterface, string, Exception> OnResultParsingError;
+
+        class ConfigurationDataBasicLoginFrenchTorrentDb : ConfigurationData

```

Comment:
Well, it has 2 javascript "features" to block http parsing :/
the first one is very complicated (weird js function with complicated input)
I first thought about using v8 to parse it but was not sure it would work with mono and not sure it would work so I decided to go with a simple way...